nn/math_.py

```diff
@@ -315,3 +315,16 @@ def cumsum(
     name=name)
   del state
   return layer
+
+
+def normalize(a: nn.Tensor, *, axes: Union[nn.Dim, Sequence[nn.Dim]], epsilon: float = 1e-6):
```

Comment:
Our convention is to use singular (`axis`), if both a single or multiple axes are possible. This is because in practice, using a singular axis is more common.

Reply:
Also return type missing.
```suggestion
def normalize(a: nn.Tensor, *, axis: Union[nn.Dim, Sequence[nn.Dim]], epsilon: float = 1e-6) -> nn.Tensor:
```